server.js

```diff
@@ -1300,12 +1300,9 @@ cache(function(data, match, sendBadge, request) {
   var user = match[1];  // eg, `symfony/symfony`.
   var branch = match[2];// eg, `/2.4.0.0`.
   var format = match[3];
-  var apiUrl = 'http://hhvm.h4cc.de/badge/' + user + '.json';
-  if (branch) {
-    // Remove the leading slash.
-    apiUrl += '?branch=' + branch.slice(1);
-  }
+  var apiUrl = 'https://php-eye.com/api/v1/package/'+user+'.json';
```

Comment:
Could you update the variable declarations to use the ES6 style `let` and `const` as applicable while you're working on this code.